06_create_cluster.sh

```diff
@@ -34,6 +34,12 @@ if [[ ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
     
     oc patch configs.samples cluster \
       --type merge --patch "{\"spec\":{\"samplesRegistry\":\"${LOCAL_REGISTRY_DNS_NAME}\"}}"
+    
+    # Some CI tests try to test scale in and out of machines. For that purpose,
+    # if NUM_EXTRA_WORKERS is set, we enable extra worker for that test.
+    if [ -f "${OCP_DIR}/extra_host_manifests.yaml" ]; then
```

Comment:
While it could make sense to apply automatically the extra manifests during the CI, I wouldn't like to have this feature for the local development, so I'd suggest to enable it only if `OPENSHIFT_CI` is set (see https://github.com/openshift/release/blob/6ef9fb413a75b8f3cc967acc9a532dc7c92b51a4/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L77)

Reply:
This check is in the ENABLE_LOCAL_REGISTRY condition and not affecting local development. Do you mean using new environment variable `OPENSHIFT_CI` (comes from CI) in dev-scripts instead using ENABLE_LOCAL_REGISTRY?

Reply:
Yes: potentially `ENABLE_LOCAL_REGISTRY` could be enabled also during local development, so I don't think it's related. The extra worker feature it's instead something independent - I think it could even be worth to have a dedicated variable setting to automatically apply the manifest, but since right now it seems just a CI requirement I suggested `OPENSHIFT_CI`

Reply:
Sometimes I would like this behavior locally (for example testing e2e deployments where the live-iso flow adds extra workers ref https://github.com/openshift-metal3/dev-scripts/pull/1193)

Could we add some new variable which controls this behavior, e.g `APPLY_EXTRA_WORKERS` or similar, then set that in the CI config?

Reply:
Seems like a good idea

Reply:
Agree. Thanks to that we will offer a new dev-script feature instead using specific to CI.